lib/usdUfe/utils/layers.h

```diff
@@ -149,6 +149,23 @@ bool isSessionLayer(
     const PXR_NS::SdfLayerHandle&           layer,
     const std::set<PXR_NS::SdfLayerRefPtr>& sessionLayers);
 
+// Verify if a stage has any dirty layer.
+enum class StageDirtyState : int
+{
+    Clean,
+    DirtyRootLayers,
+    DirtySessionLayers
```

Comment:
According to our coding guidelines enum names should be named like constants and start with `k`

Reply:
Dang! I looked at another enum for guidance and it lacked the k. I'll add them.